src/parquet/column/writer.cc

```diff
@@ -99,13 +101,15 @@ void ColumnWriter::WriteNewPage() {
   // TODO(PARQUET-590): Encodings are hard-coded
   int64_t bytes_written = pager_->WriteDataPage(num_buffered_values_,
       num_buffered_encoded_values_, definition_levels, Encoding::RLE, repetition_levels,
-      Encoding::RLE, values, Encoding::PLAIN);
+      Encoding::RLE, values, Encoding::PLAIN, page_statistics_);
   total_bytes_written_ += bytes_written;
 
   // Re-initialize the sinks as GetBuffer made them invalid.
   InitSinks();
   num_buffered_values_ = 0;
   num_buffered_encoded_values_ = 0;
+  chunk_statistics_->Merge(*page_statistics_);
+  page_statistics_->Reset();
```

Comment:
I've leafed through parquet-mr and parquet-format; it suggests that page_statistics_ is allowed to be empty (as you have here) but not null, if I'm reading things right (there's many layers to the onion). Is there any reason why we might want to let the page statistics be nullptr?